config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root to: "groups#index"
+
+  resources :groups do
+    resources :payments
+  end
```

Comment:
- [ ] Please add only necessary routes. You can do it by using `only`.

Reply:
I will do it soon. Thanks.